### ISSUE TITLE
Unexpected event receiver syntax kind

### ISSUE THREAD

**Issue by amcasey** [dotnet/roslyn#7923]
```
System.InvalidOperationException
Microsoft.CodeAnalysis.CSharp.Symbols.EventSymbol GetEventSymbol(Microsoft.CodeAnalysis.CSharp.BoundExpression, Microsoft.CodeAnalysis.CSharp.BoundExpression ByRef, Microsoft.CodeAnalysis.CSharp.CSharpSyntaxNode ByRef)
Boolean CheckEventValueKind(Microsoft.CodeAnalysis.CSharp.BoundEventAccess, BindValueKind, Microsoft.CodeAnalysis.DiagnosticBag)
Boolean CheckValueKind(Microsoft.CodeAnalysis.CSharp.BoundExpression, BindValueKind, Microsoft.CodeAnalysis.DiagnosticBag)
Microsoft.CodeAnalysis.CSharp.BoundExpression CheckValue(Microsoft.CodeAnalysis.CSharp.BoundExpression, BindValueKind, Microsoft.CodeAnalysis.DiagnosticBag)
Microsoft.CodeAnalysis.CSharp.BoundExpression BindMemberAccessWithBoundLeft(Microsoft.CodeAnalysis.CSharp.Syntax.ExpressionSyntax, Microsoft.CodeAnalysis.CSharp.BoundExpression, Microsoft.CodeAnalysis.CSharp.Syntax.SimpleNameSyntax, Microsoft.CodeAnalysis.SyntaxToken, Boolean, Boolean, Microsoft.CodeAnalysis.DiagnosticBag)
Microsoft.CodeAnalysis.CSharp.BoundExpression BindQualifiedName(Micr...
```

**Comment by amcasey** [comment]
~270 hits in 10 days

**Comment by amcasey** [comment]
From telemetry.  Apologies for the lack of type names - it was fixed after Update 1 shipped.

**Comment by amcasey** [comment]
FYI @agocke

**Comment by jaredpar** [comment]
Fixed by #8236
